zookeeper-it/src/main/java/org/apache/zookeeper/test/system/GenerateLoad.java

```diff
@@ -713,7 +711,7 @@ private static String getMode(String hostPort) throws NumberFormatException, Unk
     }
 
     private static void doUsage() {
-        System.err.println("USAGE: " + GenerateLoad.class.getName()
+        LOG.info("USAGE: " + GenerateLoad.class.getName()
```

Comment:
I would say that the usage synopsis should still go to `System.err`.  What do you think?

Reply:
I do not have a strong opinion here
@mino181295 what's your opinion ?

Reply:
I agree with @ztzg do you think it should be `System.err.println` also line 673-676?

Reply:
I'd say so.  (Sorry for missing it on the first round.)

Reply:
ah dang, i hit approve too quickly. i agree with this requested change as well.

Reply:
oh and i see that it has been addressed :)